config/vufind/EDS.ini

```diff
@@ -403,3 +403,14 @@ AU = None
 ; the LibGuides URLs if the same name is present here.
 ;url[Expensive Science Database] = https://some.url
 ;url[Cool Humanities Database] = https://another.url
+
+; Define whether to display 'Short' or 'Long' author data on the EDS search result list page.
+; - 'Short' is fetched from the RecordInfo section of a record (the 
+; 'BibRecord/BibRelationships/HasContributorRelationships/*PersonEntity/Name/NameFull' data).
+; - 'Long' is fetched from the item 'Au' group data.
+; If you do set 'ResultListPage' to 'Short' then you can define how many author names to 
+; display in AuthorNameLimit. If a record has more author names than you define in AuthorNameLimit
+; then VuFind will display the more_authors_abbrev text.
```

Comment:
We should probably note which option is the default for clarity (though if you might be inclined to change the default behavior, maybe we should wait for that decision to be made before adding the comment -- I'll just leave this note here so we don't forget one way or another).

Reply:
thanks @demiankatz , currently the default behaviour is to continue to display the long author data.

We are happy to continue that so I've added in my latest follow-up that this is the default behaviour. But we are also open to changing that. It would be good to get your thoughts on this  @berndoberknapp?

Reply:
we are happy to proceed with the default being displaying the 'long' author data. As that is in the patchsets I will resolve this thread :)

Reply:
My personal preference would be short as default, but whether we should make such a change probably depends on the target release for this pull request?

I would prefer to also have the option to use short in the detailed record display, which would require a different section name.

Reply:
thanks @berndoberknapp . i've pushed a follow-up setting the default to be 'short' author data for the result list.

Could you , @demiankatz , and @maccabeelevine please retest to ensure that Short is indeed the default on the EDS result list page now?

I will make a separate pull request for adding the option to use short on the detailed record display, under a different section name.

Reply:
perhaps it would be easiest for me to make a new pull request for adding the Short/Long config for the EDS detailed record display once this pull request has been merged. what do you think @demiankatz ?

Reply:
> Could you , @demiankatz , and @maccabeelevine please retest to ensure that Short is indeed the default on the EDS result list page now?

Looks like the default is still 'Long' in the template line 39.

Reply:
my apologies @maccabeelevine . I had not pushed my change to the EDS.ini file, uncommenting the ResultListPage setting and changing it to 'Short'. I've now pushed that change in my latest commit and that resulst-list.phtml template line 39 should now correctly default to 'Short'

Reply:
Hi, I can confirm that the "Short" behavior is now the default if I don't make any changes to EDS.ini.  Thanks!  That said there are two minor issues with the default:

- result-list.phtml line 39 still has 'Long' as the default.  That means if someone doesn't get the EDS.ini update at all, they will still have the Long behavior.  That's common if they have overridden EDS.ini into a local folder and forget to update the overridden version on upgrade.  It's also just inconsistent.
- EDS.ini says `By default, the EDS detailed record page will show the Long author data.`  Which is true in a sense given my previous comment, but I don't think it's what you mean.